pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -380,6 +383,9 @@ spec:
               mountPath: /var/run
             - name: conf
               mountPath: /usr/local/etc/istgt
+            - name: tmp
+              mountPath: /tmp
```

Comment:
@princerachit - what will be name of the core file? Will it have a pid or be dumped as istgt.dump? wondering if we need to put that into unique folder like the cache file. 

Similarly, can you update the core location for cstor pool binaries as well.

Reply:
@kmova , I have seen istgt dumping core with name "istgt.core" in the same directory where the istgt binary is present. Maybe @vishnuitta or @payes  can answer if there is any option to specify the core directory while running the istgt binary.

Reply:
@vishnuitta , please confirm if cstor-pool is dumping its core inside /tmp.

Reply:
cores will be dumped based on the content of core_pattern.. (something like https://sigquit.wordpress.com/2009/03/13/the-core-pattern/)
And, we need host storage to dump the cores.
As this being node level setting, and, cstor-pool pods having /tmp as its directory from host storage, we need to have core_pattern such that cores goes into /tmp directory.
So, /tmp directory should be mounted into cstor-istgt pods also.

Reply:
On following the steps in the above link I was able to set the core file dump location to /tmp in the pod.
Note: When pattern in `/proc/sys/kernel/core_pattern` is changed from the pod, the changes are reflected to the host node. Therefore all the dumps of processes running on the host node would be written to /tmp of host. Whenever the node restarts the core dumps of host node would be lost.